uniffi/src/lib.rs

```diff
@@ -354,6 +358,59 @@ unsafe impl ViaFfi for String {
     }
 }
 
+/// Support for passing timestamp values via the FFI.
+///
+/// Timestamps prior to the UNIX EPOCH will cause a panic.
```

Comment:
Hrm...I wonder how likely we are to encounter these panics in practice.

For most other datatypes, IIUC we should *never* encounter an invalid value, which is why we're happy to panic here. But reading e.g. the docs for [`java.time.Instant`](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html) it seems like API consumers could easily create a pre-epoch Instant on the Kotlin side and try to pass it into the Rust code.

That's probably fine, so long as the Kotlin wrapper takes responsibility for guarding against it and turning it into an exception instead of a crash. More generally: a panic should only happen due to some internal implementation bug in UniFFI, not due to the API consumer mis-using the generated code.

I don't have any concrete change to suggest here, just thinking it through out loud...